packages/components/src/components/dropdown/_dropdown.scss

```diff
@@ -65,6 +65,13 @@
     }
   }
 
+  // Menu's triggering element updated to button with Downshift v5 upgrade
+  .#{prefix}--dropdown button {
```

Comment:
Should we use this selector to target it:

```scss
.#{$prefix}--dropdown #{$prefix}--list-box__field
```

As an aside, `prefix` is missing the `$` in front 👍

Reply:
```suggestion
  .#{$prefix}--dropdown .#{$prefix}--list-box__field {
```